wagtail/admin/templates/wagtailadmin/widgets/tag_widget.html

```diff
@@ -1 +1,9 @@
-{% include 'django/forms/widgets/text.html' %}<script>initTagField('{{ widget.attrs.id|escapejs }}', '{{ widget.autocomplete_url|escapejs }}', {% if widget.tag_spaces_allowed %}true{% else %}false{% endif %});</script>
\ No newline at end of file
+{% include 'django/forms/widgets/text.html' %}
+<script>
+    initTagField(
+        '{{ widget.attrs.id|escapejs }}',
+        '{{ widget.autocomplete_url|escapejs }}',
+        {%if widget.tag_spaces_allowed %}true{% else %}false {% endif %},
+        {% if widget.tag_limit %}{{widget.tag_limit}}{%else %}null{% endif %}
```

Comment:
`%else` -> `% else`

Reply:
fixup -> 8d8081c

Thank you for taking a quick look.